### DISCUSSION TITLE
URGENT: no one has access to my profile even though I can access it

### DISCUSSION THREAD

**Discussion by Sstoica24** [community/community#116560] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello,
Even though I am able to access my account, other people are unable to see it. They get the following error: 
<img width="655" alt="Screenshot 2024-04-02 at 10 41 02 AM" src="https://github.com/community/community/assets/143836130/7a13c094-3d61-41cf-930a-e52176f5abe3">

What is going on? Originally, I thought that this was because my account may have somehow been set to private. However, it is not: 
<img width="983" alt="Screenshot 2024-04-02 at 10 42 07 AM" src="https://github.com/community/community/assets/143836130/e54eb363-8aad-4e14-a066-b3185201fcb4">

This happened right after I tried merging two branches together and deleting the one I did not want anymore. 

What should I do to fix this issue? I need people to be able to see and access my repositories.

Thank you,
Sofia

**Comment by rathaumons** [comment]
Strange!!! Happened to mine too. I had to reset my password.

By the way, I can see you just fine.

![image](https://github.com/community/community/assets/60795657/f4afb1b5-7630-40c6-9431-4f55e8132af5)

**Reply by Sstoica24** [reply]
After reseting my password for the third time I think it finally worked. Thank you so much for you help!

**Reply by rathaumons** [reply, marked as answer]
You're welcome. Anyway, it happened to many accounts. You can see the security log https://github.com/settings/security-log and we need an explanation from GitHub.

**Reply by meetkool** [reply]
hey this has happened to me too and i am support confiused now in my security log https://github.com/settings/security-log 

it is showing @github-staff@meetkool	
[github-staff ](https://github.com/settings/security-log?q=actor%3Agithub-staff)– [user.suspend](https://github.com/settings/security-log?q=action%3Auser.suspend)
Suspended access for [meetkool](https://github.com/meetkool) to GitHub

**Comment by jfullstackdev** [comment]
I can see your public profile

![image](https://github.com/community/community/assets/47092464/b8958c86-9a27-4588-ba1e-9718353c5d5f)

**Comment by 7godwin** [comment]
check docs or stackoverflow

**Comment by rohit-yadavv** [comment]
check stackOverflow you will definitely find something there
